files/demo.hledger

```diff
@@ -0,0 +1,178 @@
+﻿# Welcome to hledger, This file is where your financial history will be recorded.
+# It is a plain text file, so it is easy to edit and manipulate, and you can use
+# search and replace functionality in your text editor
+
+# Everything after '#', ';' or '*' are comments, so they are not read by hledger,
+# except in some cases to be discussed later
+
+
+# To start configuring our journal, it is a good idea to define a default currency
+D USD 1,000.00  ; Format - D Currency amountFormat
+
+
+# Now let's declare our main accounts. The order they appear here is the other they will
+# show on the reports. We need to tell hledger what is the name of each account using the
+# following code: A=Asset, L=Liabilities, R=Revenues, X=Expenses, E=Equity.
+# We will use the default name, but you can change them if you like
+
+# Attention! Here and in some other places you may need at least 2 spaces, instead of 1
+
+account Asset                  A  ; Format - account name  Code  <- 2 spaces at least between name and code
+account Liabilities            L
+account Revenue                R
+account Expenses               X
+account Equity                 E
+
+
+# Let's declare some other accounts, so they will appear in the desired order
+# You can organize them in a tree structure using ':' to indicate a a parent-child relation account
```

Comment:
s/account//  ?

Reply:
Do you mean another account separator? I would rather omit that but I can add if you think that's important

Reply:
I just meant: consider dropping the final "account" word.
"to indicate a a parent-child relation account" -> "to indicate a parent-child relation"